examples/kafka-hub/consolidator/docker/alpine/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# syntax = docker/dockerfile:1.0-experimental
+# ------------------------------------------------------------------------
+#
+# Copyright 2022 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+# ------------------------------------------------------------------------
+
+# ACR image tag: adoptopenjdk/openjdk11:jre-11.0.16_8-alpine
+ARG BASE_IMAGE_DIGEST="sha256:409b76ab70e60ca49fb974171fc40a44fffe6da89591ce80835d33ddb801b4c1"
```

Comment:
Instead of hardcoding, can we pass this at build time?

Reply:
parameterized this value. This will be passed from the pipeline.